src/main/java/org/jabref/gui/icon/InternalMaterialDesignIcon.java

```diff
@@ -11,40 +11,43 @@
 
 import org.jabref.gui.util.ColorUtil;
 
-import de.jensd.fx.glyphs.GlyphIcons;
+import org.kordamp.ikonli.Ikon;
+import org.kordamp.ikonli.javafx.FontIcon;
 
 public class InternalMaterialDesignIcon implements JabRefIcon {
 
-    private final List<GlyphIcons> icons;
+    private final List<Ikon> icons;
     private Optional<Color> color;
     private final String unicode;
 
-    public InternalMaterialDesignIcon(Color color, GlyphIcons... icons) {
+    public InternalMaterialDesignIcon(Color color, Ikon... icons) {
         this(color, Arrays.asList(icons));
     }
 
-    InternalMaterialDesignIcon(Color color, List<GlyphIcons> icons) {
+    InternalMaterialDesignIcon(Color color, List<Ikon> icons) {
         this(icons);
         this.color = Optional.of(color);
     }
 
-    public InternalMaterialDesignIcon(GlyphIcons... icons) {
+    public InternalMaterialDesignIcon(Ikon... icons) {
         this(Arrays.asList(icons));
     }
 
-    public InternalMaterialDesignIcon(List<GlyphIcons> icons) {
+    public InternalMaterialDesignIcon(List<Ikon> icons) {
         this.icons = icons;
-        this.unicode = icons.stream().map(GlyphIcons::unicode).collect(Collectors.joining());
+        this.unicode = icons.stream().map(icon->icon.getCode()).map(String::valueOf).collect(Collectors.joining());
         this.color = Optional.empty();
     }
 
     @Override
     public Node getGraphicNode() {
-        GlyphIcons icon = icons.get(0);
+
+        Ikon icon = icons.get(0);
+        FontIcon fontIcon = FontIcon.of(icon);
```

Comment:
FontIcon is already a graphical representation (extends Text), so you can simply return that. Color can be set via iconColorProperty

Reply:
Ah thanks for the hint, now it works!